### DISCUSSION TITLE
Welcome to Bug Discussions!

### DISCUSSION THREAD

**Discussion by ArcerionDev** [Prodigy-Hacking/ProdigyMathGameHacking#400] [(B) Bug Discussion]
#### Here we're going to talk about bugs & issues with hacks, and we'll try and figure them out ASAP!

Refer to our [issues](https://github.com/Prodigy-Hacking/ProdigyMathGameHacking/issues) for answers before you start a new discussion.

**Comment by NessiePUYOY** [comment]
when I go on the cheat menu, i cannot morph because it freezes my screen even if i click fix morph crash before or reload the page. Please fix this please and thank you :)

**Reply by ghost** [reply]
Same for me.

**Reply by PatheticMustan** [reply]
ah, what's the error you get? Would it be possible to screenshot it?
Also, what did you do before it crashed

**Reply by NessiePUYOY** [reply]
I dont quite remember what i did before this happened but I just click on morph player and i chose chill and char and it froze my screen so im like alright, i guess i should reload it and see what happens so i did and i did it again without clicking fix morph crash, then i l reloaded once more and then i opened the menu and then i saw it it had an option to, "fix morph crash" so i clicked fix morph crash and than i selected to morph into eclipse and my screen froze...

**Reply by ghost** [reply]
this has been reported like 5 times

**Reply by ghost** [reply]
i know it happens there isnt even an error in console

**Comment by NessiePUYOY** [comment]
Also, the levels dont save when i click on save character where it says all hacks will be saved, it does the opposite, it changes my level to a level 100 it only saves levels that are lower than 100 or 100 it self so like i cannot be a level 6000 etc.

**Reply by PatheticMustan** [reply]
Yeah, levels over 100 cause some issues with arena battling. We decided to just limit it to 100.

**Comment by NessiePUYOY** [comment]
maybe there is a way you can fix it?!

**Reply by ghost** [reply]
Yeah could y'all like make it where we can get any level we want?

**Reply by PatheticMustan** [reply]
there's a manual override, but as I said, levels beyond 100 screw up many other features (pvp, I think it also messes with titans)

**Reply by hellonearth311** [reply]
As an alternative, use the add HP and add damage. It'll give you all the benifits, but you won't have the level 100000 or somethin...

**Comment by ghost** [comment]
400th issue or pr

**Comment by ghost** [comment]
hi can someone help me my american express debit card under the name bobby fancywoman with the 12 digits of 365073526102146 and the 3 numbers are 214 and the date thing is 03/24 isnt going through on https://amazon.com or https://digitalteal.com or anything

**Reply by ghost** [reply]
i need something to do my dad is talking to his friends and drinking since 5 pm everyone's offline on discord pokefarm q is boring because i have this great way to monetize an extension but its too hard to code my mom wont let me watch the simpsons im bored i have nothing to do :(((((((((((((

**Reply by ghost** [reply]
e

**Reply by ghost** [reply]
ok ima stop now
